autofs-5.0.4 - fix lsb init script header

From: Ian Kent <raven@themaw.net>

It truns out that "Should-Start:" is not sufficient to enforce
the required ordering of services and a "Required-Start:" is
needed instead.
---

 CHANGELOG             |    1 +
 redhat/autofs.init.in |    4 ++--
 samples/rc.autofs.in  |    4 ++--
 3 files changed, 5 insertions(+), 4 deletions(-)


diff --git a/CHANGELOG b/CHANGELOG
index 91edd14..2be7683 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -35,6 +35,7 @@
 - use misc device ioctl interface by default, if available.
 - fix call restorecon when misc device file doesn't exist.
 - clear rpc client on lookup fail.
+- fix lsb init script header.
 
 4/11/2008 autofs-5.0.4
 -----------------------
diff --git a/redhat/autofs.init.in b/redhat/autofs.init.in
index a9a33c8..fded1d8 100644
--- a/redhat/autofs.init.in
+++ b/redhat/autofs.init.in
@@ -9,8 +9,8 @@
 #
 ### BEGIN INIT INFO
 # Provides: $autofs
-# Should-Start: $network $ypbind
-# Should-Stop: $network $ypbind
+# Required-Start: $network $ypbind
+# Required-Stop: $network $ypbind
 # Default-Start: 3 4 5
 # Default-Stop: 0 1 2 6
 # Short-Description: Automounts filesystems on demand
diff --git a/samples/rc.autofs.in b/samples/rc.autofs.in
index 78cbec2..b193a4e 100644
--- a/samples/rc.autofs.in
+++ b/samples/rc.autofs.in
@@ -8,8 +8,8 @@
 #
 ### BEGIN INIT INFO
 # Provides: $autofs
-# Should-Start: $network $ypbind
-# Should-Stop: $network $ypbind
+# Required-Start: $network $ypbind
+# Required-Stop: $network $ypbind
 # Default-Start: 3 4 5
 # Default-Stop: 0 1 2 6
 # Short-Description: Automounts filesystems on demand
